README.md

```diff
@@ -110,7 +110,12 @@ These script tags add the variable `ramda` on the browser's global scope.
 
 Or you can inject ramda into virtually any unsuspecting website using [the bookmarklet](BOOKMARKLET.md).
 
-### Partial Builds
+### Build
+
+* on Unix-based platforms, `npm run build` updates __dist/ramda.js__ and __dist/ramda.min.js__
+* on Windows, write the output of `scripts/build --complete` to a temporary file, then rename the file as __dist/ramda.js__.
```

Comment:
The word "as" doesn't seem right to me. Rather than

> […] rename the file as __dist/ramda.js__.

let's write

> […] rename the temporary file __dist/ramda.js__.